README.md

```diff
@@ -55,7 +55,7 @@ We suggest that you add the following to your shell configuration:
 
 ```sh
 export PATH="$PATH:~/.bloop"
-alias bloop="bloop-ng.py"
+alias bloop="~/.bloop/bloop-ng.py bloop.Cli --"
```

Comment:
Could you please remove the prefixing `~/.bloop/`? Otherwise it'd break for people that install the client on their `$PATH` but not in `~/.bloop`.